utils/omit.ts

```diff
@@ -0,0 +1,17 @@
+export function omit<
+  TObj extends Record<string, unknown>,
+  TKey extends keyof TObj,
+>(obj: TObj, ...keys: TKey[] | [TKey[]]): Omit<TObj, TKey> {
+  const actualKeys: string[] = Array.isArray(keys[0])
+    ? (keys[0] as string[])
+    : (keys as string[]);
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const newObj: any = Object.create(null);
+  for (const key in obj) {
+    if (!actualKeys.includes(key)) {
+      newObj[key] = obj[key];
+    }
+  }
+  return newObj;
+}
```

Comment:
Would be silly to include this full util to exclude one header, right @sachinraja

Reply:
Yeah that's what I was thinking too

Reply:
Could it cause some pain down the line.

a user can just explicitly delete the troubled headers. Just need to show an example so it clicks with people
```ts
export default withTRPC<AppRouter>({
  config({ ctx }) {

     let headers = ctx?.req?.headers
     delete headers?.connection
```